### PR TITLE
Milestone -1 

### PR DESCRIPTION
**Pull request by den-cloud**
In this feature, we will add the following:

Add the basic Html structure for the Awesome Books webpage.
Create an Array that keeps a collection of book objects.
Create a function to add a new book to the collection.
Create a function to remove a book from the collection.
Display all books saved in the collection in the top part of the page.
Data is preserved in the browser's memory by using localStorage.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+form.addEventListener('submit', handleForm);
+
+function renderBookList() {
+  const booksListContainer = document.getElementById('book-list-container');
+
+  const p = document.createElement('P');
+  const pText = document.createTextNode(books[books.length - 1]);
+  const removeButton = document.createElement('Button');
+  const buttonText = document.createTextNode('Remove');
+  const line = document.createElement('Hr');
```
Comment (dasileker) [inline app.js]: - [x] Please this function should follow the ES6 rule for arrow function.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  booksListContainer.appendChild(p);
+  removeButton.appendChild(buttonText);
+  removeButton.setAttribute('onclick', `removeBook(${[totalBooks - 1]})`);
+  booksListContainer.appendChild(removeButton);
+  booksListContainer.appendChild(line);
+}
+
+function addNewBook() {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
```
Comment (dasileker) [inline app.js]: - [x] This function  for adding new book should also follow the ES6 rule too.
- [ ] Optional you can concat the Title and Author here to make you adding book look good

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @den-cloud,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

- Please Add details to your PR.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**